src/routes/play/+page.svelte

```diff
@@ -209,9 +210,10 @@
 						bind:this={textareaRef}
 						bind:value={prompt}
 						on:click={toggleSuggestions}
+						on:keydown="{(event) => event.key === 'Enter' && buttonRef.click()}"
 						disabled={isLoading}
 					/>
-					<button class="chat__button-generate" type="submit" disabled={!prompt}>
+					<button class="chat__button-generate" type="submit" disabled={!prompt} bind:this={buttonRef}>
```

Comment:
Can't have the `keydown` event submit the form because it's implemented as `<form use:enhance>`, so we trigger a click event on the existing submit button.